<?xml version="1.0"?>
<launch>
  
  <!-- Hardware Interface -->
  <node name="gazebo_hardware_interface" pkg="grips_control" type="gazebo_hardware_interface.py" output="screen"> 
    <param name="write_ip" type="string" value="192.168.0.4"/>
    <param name="read_port" type="int" value="5051"/>
    <param name="write_port" type="int" value="5052"/>
    <param name="publish_rate" type="double" value="500.0" />
  </node>
  
</launch>
